setup.py

```diff
@@ -34,6 +34,10 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
What about 3.7?

Reply:
breaks on travis 🤦‍♂ 
https://travis-ci.org/opentracing-contrib/python-django/jobs/540486339
<img width="1416" alt="Screenshot 2019-06-02 14 17 58" src="https://user-images.githubusercontent.com/1268088/58767526-3b8d5f80-8541-11e9-8c90-36c3156fdb3b.png">

Reply:
You can add `dist: xenial` to `.travis.yml` to solve it.